### DISCUSSION TITLE
Release 0.11.5 with support of newest pythons

### DISCUSSION THREAD

**Discussion by ivandasch** [confluentinc/ducktape#411] [General]
Hi folks! After merging #397 it seems there is about time to ship this feature. Nowadays it's quite hard to find distros with older pythons installed by default. Do you have any plan about the next release?

**Comment by nizhikov** [comment]
+1

**Comment by anton-vinogradov** [comment]
+1

**Comment by wernerdv** [comment]
+1

**Comment by skorotkov** [comment]
+1

**Comment by nao-it** [comment]
+1

**Comment by vveider** [comment]
+1

**Comment by shishkovilja** [comment]
+1

**Comment by btwokr** [comment]
+ 1

**Comment by NSAmelchev** [comment]
+1

**Comment by imcdo** [comment]
+1, let me see if I can ask the team if we can cut a release.

**Comment by ivandasch** [comment]
@imcdo Is there any news regarding the new release?

**Comment by nagpavan-chilakam** [comment]
Hey Community members,
Firstly sorry for delays.
This is on our mind.
We are doing some internal testing.
We will get back when we can do the release in next few days.

**Comment by ivandasch** [comment]
@nagpavan-chilakam Hi, is there any news? I don't want to annoy you, but EOL of python 3.8 is approaching...

**Comment by imcdo** [comment]
@ivandasch  the release process has begun. If you wish to try it out you can pull this tag https://github.com/confluentinc/ducktape/tree/v0.12.0a941. 
Expect the release in the next week, as @nagpavan-chilakam has completed the testing for it (thanks!)

**Comment by nagpavan-chilakam** [comment]
@ivandasch we have published the new release 0.12.0 https://pypi.org/project/ducktape/
https://github.com/confluentinc/ducktape/tree/v0.12.0
Doc updates is pending, which I will inform once done, but the release should be available for consumption now

**Comment by nagpavan-chilakam** [comment]
Hello All, we are happy to announce new release v0.12.0
https://github.com/confluentinc/ducktape/discussions/418

**Comment by ivandasch** [comment]
Hurray!!!!
